### DISCUSSION TITLE
Child Theme PS 8.0.1 + Modules 8.0.1

### DISCUSSION THREAD

**Discussion by lanaworks** [PrestaShop/PrestaShop#31535] [Q&A]
Hi,

PrestaShop 8.0.1
PHP 7.4.x / 8.1.x

Since version 8.0.x of PrestaShop, I have been confronted with many bugs with child themes:
- translations not working
- overides (assets) do not work
- some modules cannot be installed

These issues have been known for several months and there is still no fix.

Is the opensource PrestaShop project still maintained?

Best regards

**Comment by kpodemski** [comment]
Hello @lanaworks 

Could you be more precise? You wrote: "These issues have been known for several months and there is still no fix.". Do you have any links to the added [issues](https://github.com/PrestaShop/PrestaShop/issues)?

You also wrote that "some modules cannot be installed", how is that related to the child theme feature? 🤔 

> Is the opensource PrestaShop project still maintained?

I think you know the [answer](https://github.com/PrestaShop/PrestaShop/pulse), no need to be mean.

**Reply by lanaworks** [reply]
Hi @kpodemski ,

Child theme translation :
- https://github.com/PrestaShop/PrestaShop/issues/30716
- https://www.sunnytoo.com/forums/topic/translations-not-working#post-144862

Regarding "some modules cannot be installed", I have just modified the title of the subject.

The launch of PrestaShop 1.7 was a nightmare and it seems that version 8 of PrestaShop is following the same path.
PrestaShop 1.7 had at least the advantage of bringing frequent fixes...

Best

**Reply by matks** [reply]
> The launch of PrestaShop 1.7 was a nightmare and it seems that version 8 of PrestaShop is following the same path.
PrestaShop 1.7 had at least the advantage of bringing frequent fixes...

Hey @lanaworks please check our Code of Conduct https://github.com/PrestaShop/PrestaShop/blob/2340c7979944e7298d5af06b84db7d43386cce48/CODE_OF_CONDUCT.md

Passive-aggressive sentences like "Is the opensource PrestaShop project still maintained?" or "PrestaShop 1.7 had at least the advantage of bringing frequent fixes..." are against our Code of Conduct.

In this space we focus on working together on the PrestaShop project. We are not here to listen to rants and passive-aggressive sentences that bring no value to the project.

**Reply by kpodemski** [reply]
@lanaworks 

I see that a guy from SunnyToo fixed the issue. It would be best for everyone for them to contribute to the core, so everyone could have this fixed. I wasn't aware of this issue, how could I? :)

I don't know if you noticed, but a bug was there on PrestaShop 1.7.8.8, and it has nothing to do with the release of 8.0. If you want a better start for PrestaShop 8 you can help us, you can report bugs, you can help us get them fixed, you can help us translate the software. There are numerous ways how [you can get involved](https://www.prestashop-project.org/get-involved/).

**Reply by lanaworks** [reply]
> Passive-aggressive sentences like "Is the opensource PrestaShop project still maintained?" or "PrestaShop 1.7 had at least the advantage of bringing frequent fixes..." are against our Code of Conduct.

You cry because I report on the project "discussions" part that this version 8 is still a beta solution... OMG, I'm so sorry... 
After 14 years of using PrestaShop, it's time for me to say goodbye and guide my customers towards more stable solutions.
Keep up the good work...and good luck
